hw/w4/routes/cohorts.js

```diff
@@ -0,0 +1,138 @@
+const express = require("express");
+const router = express.Router();
+const knex = require("../db/clients");
+const faker = require('faker');
+const knexfile = require("../knexfile");
+const { application } = require("express");
+
+let num = 400;
+
+
+router.get("/new", (req, res) => {
+    res.render("cohorts/new");
+});
+
+//'id'
+//'member'
+//'name'
+//'logoUr;
+router.get("/", (req, res, next) => {
+  const imageUrl = "http://placeimg.com/640/"
+  num++
+  const logoUrl = req.query.logoUrl || imageUrl+num.toString()
+  const cohort_name = req.query.name || undefined
+  names = []
+  if(cohort_name !== undefined) {
```

Comment:
The problem with this condition is that it will insert data into `cohorts` table on refresh.